bears/vcs/git/GitCommitBear.py

```diff
@@ -291,6 +291,9 @@ def check_issue_reference(self, body,
         if host not in self.SUPPORTED_HOST_KEYWORD_REGEX:
             return
 
+        if not re.search(self.SUPPORTED_HOST_KEYWORD_REGEX[host], body):
+            return
```

Comment:
Yeah, that's the main issue here.
With coala and other repos, we won't have all the commits fixing some bugs, right?
And if we enforce this in `.coafile`, we would require all commits to have issue references.
That is clearly impossible.
So, a check for presence of the keywords tells us that this isn't a commit that fixes/closes something, right? Hope you get the point.